style.css

```diff
@@ -313,26 +371,306 @@ h4{
   align-items: center;
 }
 
+.background-section {
+  background: #fff;
+}
+
 .contact-section {
-  background: #6070FF;
-  background-image: url(src/img/contact\ form\ shapes-mobile@2x.png);
+  background: #6070ff;
+  background-image: url(src/img/contact\ form\ shapes-mobile@2x.svg);
   background-repeat: no-repeat;
   background-size: 55%;
   background-position-x: right;
-  background-position-y: -25%;
+  background-position-y: 2.5em;
   border-top-left-radius: 4rem;
 }
 
 .contact-text-section {
   display: flex;
   grid-column: 1/9;
   flex-direction: column;
-  border: 1px solid #dfe1e6;
-  border-top-right-radius: 4em;
+  align-items: center;
 }
 
-.contact-title h4{
-  margin: 2.5em 0.6em 0 0.6em;
+.contact-title h4 {
+  margin: 3em 0.6em 0 0.6em;
   font-size: 2.5em;
   color: #fff;
-}
\ No newline at end of file
+}
+
+form {
+  display: flex;
+  grid-column: 1/9;
+  flex-direction: column;
+  align-items: left;
+  margin: 0.6em 3em 0 0.6em;
+}
+
+.input-box input {
+  width: 100%;
+  border: 1px solid #cfd8dc;
+  border-radius: 0.5rem;
+  margin: 0.6em 1em 0 0.6em;
+  text-align: left;
+  padding: 3%;
+}
+
+.input-box textarea {
+  width: 100%;
+  border: 1px solid #cfd8dc;
+  border-radius: 0.5rem;
+  margin: 0.6em 0.6em 0 0.6em;
+  text-align: left;
+  padding: 3%;
+}
+
+button {
+  background-color: #fff;
+  padding: 3%;
+  font-weight: 700;
+  font-size: 1.06em;
+  color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 0.5rem;
+  cursor: pointer;
+  margin: 1em 0.6em 2.5em 0.6em;
+}
+
+@media (min-width: 768px) {
+  body {
+    font-size: 20px;
+  }
+
+  .logo {
+    margin: 0 2% 0 70%;
+  }
+
+  .nav-right img {
+    display: none;
+  }
+
+  .flex-div ul {
+    display: flex;
+    justify-content: space-between;
+    margin: 1em 0 0.8em 60%;
+  }
+
+  .flex-div li {
+    margin-right: 1em;
+    cursor: pointer;
+    color: #344563;
+  }
+
+  .main-container {
+    background-image: url(src/img/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-position-y: 2em;
+  }
+
+  .greeting-text h1 {
+    margin: 20% 0.6em 0 15%;
+  }
+
+  .greeting-text p {
+    margin: 0.8em 20% 0 15%;
+  }
+
+  .connect p {
+    margin: 0.8em 20% 0 15%;
+  }
+
+  .media-icon {
+    margin: 0.8em 20% 20% 15%;
+    width: 25%;
+  }
```

Comment:
- [ ] Kindly stick with the *Figma* design as closely as possible, it seems like your design is not matching with original. Please have a look at the below comparison table to understand the problem in a better way through visualization, according to which your **social media** icon has too much gap on bigger screen sizes like > 1024px. Please make it a full desktop responsive design.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-10 13-26-47](https://github.com/JhonatanErosa/Project-2_Portfolio/assets/66476329/38cf4072-efd3-4294-9731-93d406dfed2a)


</td>

<td>

![Screenshot from 2023-06-10 13-26-40](https://github.com/JhonatanErosa/Project-2_Portfolio/assets/66476329/f167a321-31d2-4386-a65d-6d24717408fe)


</td>
</tr>
</table>


#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)
![Screenshot from 2023-06-10 13-55-49](https://github.com/JhonatanErosa/Project-2_Portfolio/assets/66476329/e5edbd52-63c9-4a93-a82c-c7b4fe7f7026)